pkg/controller/pxc/replication.go

```diff
@@ -231,14 +231,21 @@ func (r *ReconcilePerconaXtraDBCluster) reconcileReplication(ctx context.Context
 		}
 	}
 
+	authPlugin, err := primaryDB.ReadVariable("default_authentication_plugin")
```

Comment:
This was done just because I didn't want to query this variable for each replication channel, but frankly, this is not that expensive and they will not like hundreds of replication channels configured.

Because of that, I think it would be to query this for every channel, which will make this much cleaner, it can be done inside `db.StartReplication` method it self.

@egegunes @pooknull wdyt?